news/tabcomplete_for_click_apps.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** 
+
+* Tab completion xontrib for python applications based on click framework.
```

Comment:
Please rephrase this to make it more clear that this is a new xontrib and what that xontrib is called.